src/inverseRobot.js

```diff
@@ -1,13 +1,20 @@
 'use strict';
 
-/*
+/**
  * @param {object} robot
- *
- * @return {object}
-*/
-
+ * @return {object | null}
+ */
 function inverseRobot(robot) {
-  // write code here
+  const inversedRobot = {};
+
+  for (const key in robot) {
+    if (inversedRobot.hasOwnProperty(robot[key])) {
```

Comment:
It's better to use 'Object.prototype.hasOwnProperty.call(inversedRobot, robot[key])' instead of 'inversedRobot.hasOwnProperty(robot[key])' to check if a property exists. This ensures the check works even if a property named 'hasOwnProperty' exists on the object.